### PR TITLE
View/searchadvancetab

### PR DESCRIPTION
**Pull request by TheYisusGithub**

### CODE REVIEW

File: src/main/java/com/fundation/logic/view/AdvancedSearchPanel.java [hunk 1]
```diff
@@ -26,155 +26,162 @@
...
+//        add(lblChannels, gbc_lblChannels);
+//
+//        JComboBox comboBoxAudioChannels = new JComboBox();
+//        GridBagConstraints gbc_comboBoxAudioChannels = new GridBagConstraints();
+//        gbc_comboBoxAudioChannels.insets = new Insets(0, 0, 5, 0);
+//        gbc_comboBoxAudioChannels.fill = GridBagConstraints.HORIZONTAL;
+//        gbc_comboBoxAudioChannels.gridx = 1;
+//        gbc_comboBoxAudioChannels.gridy = 8;
+//        add(comboBoxAudioChannels, gbc_comboBoxAudioChannels);
+//
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/AdvancedSearchPanel.java]: Delete comments, there are a lot of comments lines

File: src/main/java/com/fundation/logic/view/TableResult.java [hunk 2]
```diff
@@ -26,16 +26,22 @@
      * Initializes a TableResult instance with headers
      */
     public TableResult(){
-        model = new DefaultTableModel(new Object[] { "Path", "Extension","Size","Date","Attributes" }, 0);
-        model.addRow(new Object[]{"FILE","EXTENSION","SIZE","DATE","ATTRIBUTE"});
+        model = new DefaultTableModel(new Object[] {"Path","File", "Extension","Size","Creation_Date","Modification_Date","Last_Access_Date","Attributes"}, 0) {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/TableResult.java]: Creation_Date, Modification_Date, Last_Access_Date variable names ar not according on our code conventions.
They should be for example: creationDate, modificationDate, lastAccessDate.

File: src/main/java/com/fundation/logic/view/AdvancedSearchPanel.java (hunk 1 above)
Comment (TheYisusGithub) [reply src/main/java/com/fundation/logic/view/AdvancedSearchPanel.java]: Oks that was remove.

File: src/main/java/com/fundation/logic/view/TableResult.java (hunk 2 above)
Comment (TheYisusGithub) [reply src/main/java/com/fundation/logic/view/TableResult.java]: OK that was fixed.

File: src/main/java/com/fundation/logic/view/AdvancedAudioPanel.java [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+     * The constructor let initializer component.
+     */
+    public AdvancedAudioPanel() {
+        initComponent();
+    }
+
+    /**
+     * This method set the panel and show the content panel audio.
+     */
+    public void initComponent(){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedAudioPanel.java]: put space between ")" and "{"

File: src/main/java/com/fundation/logic/view/AdvancedAudioPanel.java [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+    /**
+     * This method return the JButton related btnSearch Advance Audio.
+     * @return btnSearchAdvanceAudio
+     */
+    public JButton getBtnSearchAdvanceAudio() {
+        return btnSearchAdvanceAudio;
+    }
+
+    /**
+     * This method return comboBoxAudioCodecName
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedAudioPanel.java]: Put dat at the end of the code

File: src/main/java/com/fundation/logic/view/AdvancedAudioPanel.java [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+    /**
+     * This method return comboBoxAudioChannel
+     * @return comboBoxAudioChannel
+     */
+    public JComboBox getComboBoxAudioChannel() {
+        return comboBoxAudioChannel;
+    }
+
+    /**
+     * This method return comboBoxAudioSampleRate
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedAudioPanel.java]: put dat "."at the end of the code

File: src/main/java/com/fundation/logic/view/AdvancedImagePanel.java [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.view;
+
+import com.fundation.logic.view.components.SettingPanel;
+
+import javax.swing.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedImagePanel.java]: Put specific library

File: src/main/java/com/fundation/logic/view/AdvancedImagePanel.java [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+import java.awt.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedImagePanel.java]: specific library

File: src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java [hunk 8]
```diff
@@ -0,0 +1,236 @@
...
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.view;
+
+import com.fundation.logic.view.components.SettingPanel;
+import javax.swing.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java]: Put specific library

File: src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java [hunk 9]
```diff
@@ -0,0 +1,236 @@
...
+import java.awt.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java]: Put specific library

File: src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java [hunk 10]
```diff
@@ -0,0 +1,236 @@
...
+        btnSearchAdvanceVideoPanel = new JButton("Search");
+        GridBagConstraints gbc_btnSearch = new GridBagConstraints();
+        gbc_btnSearch.gridwidth = 2;
+        gbc_btnSearch.gridx = 0;
+        gbc_btnSearch.gridy = 14;
+        add(btnSearchAdvanceVideoPanel, gbc_btnSearch);
+    }
+
+    /**
+     * This method return comboBoxVideoCodec
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java]: Put dat "." at the end of code

File: src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java [hunk 11]
```diff
@@ -0,0 +1,236 @@
...
+    /**
+     * This method return comboBoxAudioCodecName
+     * @return comboBoxAudioCodecName
+     */
+    public JComboBox getComboBoxAudioCodecName() {
+        return comboBoxAudioCodecName;
+    }
+
+    /**
+     * This method return btnSearchAdvanceVideoPanel
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java]: Put dat "." at the end of code

File: src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java [hunk 12]
```diff
@@ -0,0 +1,236 @@
...
+    /**
+     * This method return textFieldDurationFirst
+     * @return textFieldDurationFirst
+     */
+    public JTextField getTextFieldDurationFirst() {
+        return textFieldDurationFirst;
+    }
+
+    /**
+     * This method return comboBoxAudioCodecName
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java]: Put dat "." at the end of code

File: src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java [hunk 13]
```diff
@@ -0,0 +1,236 @@
...
+    /**
+     * This method return textFieldDurationsSecond
+     * @return textFieldDurationsSecond
+     */
+    public JTextField getTextFieldDurationsSecond() {
+        return textFieldDurationsSecond;
+    }
+
+    /**
+     * This method return textFieldDurationFirst
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java]: Put dat "." at the end of code

File: src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java [hunk 14]
```diff
@@ -0,0 +1,236 @@
...
+    /**
+     * This method return comboBoxResolution
+     * @return comboBoxResolution.
+     */
+    public JComboBox getComboBoxResolution() {
+        return comboBoxResolution;
+    }
+
+    /**
+     * This method return textFieldDurationsSecond
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java]: Put dat "." at the end of code

File: src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java [hunk 15]
```diff
@@ -0,0 +1,236 @@
...
+    /**
+     * This method return comboBoxVideoFramerate
+     * @return comboBoxVideoFramerate
+     */
+    public JComboBox getComboBoxVideoFramerate() {
+        return comboBoxVideoFramerate;
+    }
+
+    /**
+     * This method return comboBoxResolution
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java]: Put dat "." at the end of code

File: src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java [hunk 16]
```diff
@@ -0,0 +1,236 @@
...
+    /**
+     * This method return comboBoxVideoBitrate
+     * @return comboBoxVideoBitrate
+     */
+    public JComboBox getComboBoxVideoBitrate() {
+        return comboBoxVideoBitrate;
+    }
+
+    /**
+     * This method return comboBoxVideoFramerate
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java]: Put dat "." at the end of code

File: src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java [hunk 17]
```diff
@@ -0,0 +1,236 @@
...
+    /**
+     * This method return comboBoxVideoCodec
+     * @return comboBoxVideoCodec
+     */
+    public JComboBox getComboBoxVideoCodec() {
+        return comboBoxVideoCodec;
+    }
+
+    /**
+     * This method return comboBoxVideoBitrate
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/AdvancedVideoPanel.java]: Put dat "." at the end of code

File: src/main/java/com/fundation/logic/view/TableResult.java [hunk 18]
```diff
@@ -26,16 +26,22 @@
      * Initializes a TableResult instance with headers
      */
     public TableResult(){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/TableResult.java:32]: put a space before {

File: src/main/java/com/fundation/logic/view/components/BasicSearch.java [hunk 19]
```diff
@@ -0,0 +1,152 @@
...
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.view.components;
+
+import javax.swing.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/components/BasicSearch.java]: Put specific library

File: src/main/java/com/fundation/logic/view/components/BasicSearch.java [hunk 20]
```diff
@@ -0,0 +1,152 @@
...
+import java.awt.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/components/BasicSearch.java]: Put specific library

File: src/main/java/com/fundation/logic/view/components/DateSearch.java [hunk 21]
```diff
@@ -0,0 +1,121 @@
...
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.view.components;
+
+import com.toedter.calendar.JDateChooser;
+import javax.swing.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/components/DateSearch.java]: Put specific library

File: src/main/java/com/fundation/logic/view/components/DateSearch.java [hunk 22]
```diff
@@ -0,0 +1,121 @@
...
+import java.awt.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/components/DateSearch.java]: Put specific library

File: src/main/java/com/fundation/logic/view/SearchTabs.java [hunk 23]
```diff
@@ -8,13 +8,16 @@
  */
 package com.fundation.logic.view;
 
-import javax.swing.JTabbedPane;
-import javax.swing.JPanel;
+import com.fundation.logic.view.components.BasicSearch;
+import com.fundation.logic.view.components.SplitPanelSearch;
+import com.fundation.logic.view.components.SplitPanelDate;
+
+import javax.swing.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/SearchTabs.java:15]: Change * to specific library

**Pull request merged**